nearup

```diff
@@ -46,11 +51,11 @@ if not os.path.exists(main_script):
         add_path_if_exist('~/.zprofile')
         add_path_if_exist('~/.config/fish')
         add_path_if_exist('~/.bash_profile')
-        print(
-            '''To get started you need Nearup's directory ($HOME/.nearup) in your PATH
-environment variable. Next time you log in this will be done automatically.
+        print(STARTUP_MESSAGE)
+else:
```

Comment:
this indentation looks incorrect to me. Can you check whether it runs?

Reply:
The indentation is correct, this `else` is for the `if` at line 21 (in this PR) `if not os.path.exists(main_script):
`